### PR TITLE
ui design

### PR DESCRIPTION
**Pull request by mim24sd**
This pull request is for html and css of movie box

### CODE REVIEW

File: assests/styleSheets/general.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 0;
+}
+
+p,
+h1,
+h2,
+span {
+  font-family: var(--primaryFont);
```
Comment (takrami) [inline assests/styleSheets/general.css]: why just for these elements? it's better to add it to the body.

File: assests/styleSheets/variables.css [hunk 2]
```diff
@@ -18,5 +18,5 @@
   --space-xl: calc(3 * var(--space-unit));
 
   /* fonts */
-  --primaryFont: cursive;
+  --primaryFont: "Oxygen", sans-serif;
```
Comment (takrami) [inline assests/styleSheets/variables.css]: please add roboto font for this project and use it.

File: assests/styleSheets/indexStyle.css [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@import "./home.css";
+@import "./general.css";
```
Comment (takrami) [inline assests/styleSheets/indexStyle.css]: i think you need to add reset CSS here.

File: index.html [hunk 4]
```diff
@@ -5,6 +5,115 @@
     <link rel="stylesheet" href="./assests/styleSheets/indexStyle.css" />
   </head>
   <body>
-    <script type="text/javascript" src="./assests/javaScript/index.js"></script>
+    <main>
+      <header><h1 class="web-title">Movies</h1></header>
+      <div class="movie-box-container">
```
Comment (takrami) [inline index.html]: you can use ul here.

File: index.html [hunk 5]
```diff
@@ -5,6 +5,115 @@
...
+        <div class="movie-box">
```
Comment (takrami) [inline index.html]: each item can be a li.

File: assests/styleSheets/home.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+.movie-box-detail {
+  color: var(--lightColor);
+  text-size-adjust: auto;
+  position: absolute;
+  top: 80%;
+  left: 10%;
+  text-align: left;
+}
+
+@media screen and (min-device-width: 450px) and (max-device-width: 820px) {
```
Comment (takrami) [inline assests/styleSheets/home.css]: why do you need a media query? I think you should be able to handle the responsiveness with CSS grid.

**Review by takrami — CHANGES_REQUESTED** [review]
Very well structured and nice PR :+1:  well done :) 
Just a few comments that you need to resolve. 

andddd one OPS :D I guess you forgot to add the .gitignore file.

File: assests/styleSheets/general.css (hunk 1 above)
Comment (mim24sd) [reply assests/styleSheets/general.css]: you are right , I added it to `body`

File: assests/styleSheets/variables.css (hunk 2 above)
Comment (mim24sd) [reply assests/styleSheets/variables.css]: Sure Thanks ^^

File: index.html (hunk 4 above)
Comment (mim24sd) [reply index.html]: Such a great idea

File: index.html (hunk 5 above)
Comment (mim24sd) [reply index.html]: done

File: assests/styleSheets/home.css (hunk 6 above)
Comment (mim24sd) [reply assests/styleSheets/home.css]: after your description I changed all to 300px ^_^
Thanks , you make me to see and think better :*

File: index.html [hunk 7]
```diff
@@ -5,6 +5,93 @@
     <link rel="stylesheet" href="./assests/styleSheets/indexStyle.css" />
   </head>
   <body>
-    <script type="text/javascript" src="./assests/javaScript/index.js"></script>
+    <main>
+      <header><h1 class="web-title">Movies</h1></header>
+      <ul class="movie-box-container">
+        <a href="./movieDetail.html">
```
Comment (takrami) [inline index.html]: the first element after `ul` has to be `li`. it cannot be `a`. you can put `a` tag inside `li`.

File: assests/styleSheets/variables.css [hunk 8]
```diff
@@ -18,5 +18,5 @@
   --space-xl: calc(3 * var(--space-unit));
 
   /* fonts */
-  --primaryFont: cursive;
+  --primaryFont: "Roboto", sans-serif;
```
Comment (takrami) [inline assests/styleSheets/variables.css:27]: Is this working? I cannot find the place where you add it to the project.

File: assests/styleSheets/indexStyle.css (hunk 3 above)
Comment (mim24sd) [reply assests/styleSheets/indexStyle.css]: such a stupid mistake I did :/
thankssss ^^

File: index.html (hunk 7 above)
Comment (mim24sd) [reply index.html]: yes after I validate the html in html validator I got and fix that.
I added this after I made other page so I forgot to remove it from this commit.

File: assests/styleSheets/variables.css (hunk 8 above)
Comment (mim24sd) [reply assests/styleSheets/variables.css:27]: maybe I made a mistake but I remember that I added it as ### font-family

File: movieDetail.html [hunk 9]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Movie Box</title>
+    <link rel="stylesheet" href="./assests/styleSheets/movieDetail.css" />
+  </head>
+  <body>
+    <main>
+      <div class="details-container">
```
Comment (takrami) [inline movieDetail.html]: you can remove this `div` and add the class to `main`.

File: movieDetail.html [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+        <div class="cover-image-and-webtitle">
+          <h1>Movies</h1>
+          <img
+            src="./assests/images/luca-cover.jpeg"
+            alt="luca animation cover image"
+            class="cover-image"
+          />
+        </div>
+        <div class="detail-text">
```
Comment (takrami) [inline movieDetail.html]: why is this div needed? can't we remove it?

File: movieDetail.html [hunk 11]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Movie Box</title>
+    <link rel="stylesheet" href="./assests/styleSheets/movieDetail.css" />
+  </head>
+  <body>
+    <main>
+      <div class="details-container">
+        <div class="cover-image-and-webtitle">
```
Comment (takrami) [inline movieDetail.html]: can't we avoid this div?

File: movieDetail.html (hunk 9 above)
Comment (mim24sd) [reply movieDetail.html]: they both have their own class :(

Comment (mim24sd) [reply movieDetail.html]: this separation is for mobile

File: movieDetail.html (hunk 11 above)
Comment (mim24sd) [reply movieDetail.html]: I tried but it's children are setting as row without it

File: movieDetail.html (hunk 10 above)
Comment (mim24sd) [reply movieDetail.html]: it's for styling all text part.
in mobile changed it to `section` cause without this ,it's children are setting as row

File: movieDetail.html (hunk 9 above)
Comment (mim24sd) [reply movieDetail.html]: they both have their own design and without it I can not handle the UI design :(

File: movieDetail.html [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+    <figure class="main-container" id="movie-detail">
+      <img
+      src="./assests/images/luca-cover.jpeg"
+      alt="Movie cover image" class="cover-image" />
+      <figcaption class="detail-text">
+        <section class="detail-text-header">
+          <h2 class="movie-title">Luca</h2>
+          <p class="movie-rate">7/10</p>
+        </section>
+        <summary class="movie-description">
```
Comment (takrami) [inline movieDetail.html]: `summary` has to be a child of `details`. I suggest changing it to `p` and moving it inside the section.

Comment (mim24sd) [reply movieDetail.html]: thanksssss

**Comment by mim24sd** [comment]
ui design

**Pull request merged**